src/UserFrontend.php

```diff
@@ -90,7 +90,7 @@ public static function login_form_defaults(array $args) {
    */
   public static function login_enqueue_scripts() {
     if (file_exists(get_stylesheet_directory() . '/dist/styles/login.css')) {
-      wp_enqueue_style('theme/login', get_stylesheet_directory_uri() . '/dist/styles/login.css');
+      wp_enqueue_style('theme/login', get_stylesheet_directory_uri() . '/dist/styles/login.css', FALSE);
```

Comment:
(and elsewhere) The 3rd argument to https://developer.wordpress.org/reference/functions/wp_enqueue_style/ is an array of dependencies. The 4th argument would be the version, but it already defaults to `FALSE`, so no need to specify it manually.